source/_posts/2017-01-25-on-aggregates-and-external-context-interactions.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: On Aggregates and their interaction with the external world
+category: PHP
+tags: ["php", "ddd", "cqrs", "event sourcing", "aggregate", "aggregate root", "patterns", "clean code"]
+year: 2017
+month: 1
+day: 25
+published: true
+summary: IDDD/DDD Aggregates often have to talk to the external world somehow: a practical approachs
+description: A practical example of how DDD Aggregates can talk to the external world without the need to "know" about its 
```

Comment:
I think this is my keyboard gone AWOL - it's a bit broken, and sometimes it just hangs. Pushed, thanks :-)